README.md

```diff
@@ -14,9 +14,19 @@ See the [oci-volume-provisioner][3] for more information.
 We publish the OCI flexvolume driver as a single binary that needs to be
 installed on every node in your Kubernetes cluster.
 
+### Kubernetes DaemonSet Installer
+
+The recommended way to install the driver is through the new daemonset installer mechanism.
+
+```
+kubectl apply -f https://github.com/oracle/oci-flexvolume-driver/releases/download/${flexvolume_driver_version}/oci-flexvolume-driver.yaml
+```
+
+You'll still need to add the config file as per below (we'll fix that with Instance Principals support soon).
+
 ### Ansible
 
-The recommended way to install the driver is with Ansible.
+The old way to install the driver is with Ansible.
```

Comment:
Tempted to just drop the Ansible support entirely and only recommend DS deployment but we could do that in a subsequent PR.